css/style.css

```diff
@@ -1031,21 +1035,18 @@ footer a:focus {
   #services .round-image img {
     margin-right: 25%;
   }
+}}
+
+div.col .footerheading{
+  color: #daf0f8;
 }
 
-/* @media screen and (max-width: 650px) {
-  .nav-links {
-     display: none;
-  }
-  .logo {
-    width: 60px;
-  }
-  .hamburger {
-    display: flex;
-  }
+.login-button{
+  font-size: 15px;
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;

```

Comment:
Instead of writing `padding-left` and `padding-right` sapretly, You can write `padding: 0 1.5em;`

```diff
- padding-left: 1.5rem;
- padding-right: 1.5rem;
+ padding: 0 1.5rem;
```